packages/forms-web-app/src/middleware/check-path-allowed.js

```diff
@@ -14,7 +14,8 @@ const checkPathAllowed = (req, res, next) => {
 		'/appeal-householder-decision/email-address-confirmed',
 		'/appeal-householder-decision/confirm-email-address',
 		'/appeal-householder-decision/list-of-documents',
-		'/appeal-householder-decision/application-saved'
+		'/appeal-householder-decision/application-saved',
+		'/full-appeal/submit-appeal/enter-code/*'
```

Comment:
Is the wildcard syntax correct here? Given that the items in the array are compared with the original URL using `includes()`, I'd have thought the format for that URL would be `/full-appeal/submit-appeal/enter-code` without the trailing `/*`.

Reply:
It was not working without it. There's the appeal id coming up

Reply:
Ok. Fair enough. Should there be a similar URL for householder appeals?

Reply:
I've removed this now in favour of using a modified `skipMiddlewareForPaths` middleware which skips `checkPathAllowed` for the specified routes